src/AlohaKit/Controls/NumericUpDown/NumericUpDown.cs

```diff
@@ -48,6 +48,38 @@ public Color Color
             set { SetValue(ColorProperty, value); }
         }
 
+        public static readonly BindableProperty ColorMinusProperty =
```

Comment:
Could we rename it to MinimumColor and MaximumColor?. Also, could be better if we use a Brush instead a Color!